apps/token-e2e/src/integration/view/wallet-vega.cy.js

```diff
@@ -170,7 +170,7 @@ context('Vega Wallet - verify elements on widget', function () {
       });
     });
 
-    it('should have wallet name visible', function () {
+    it.skip('should have wallet name visible', function () {
```

Comment:
why do these need to be skipped?

Reply:
@dexturr With the recent API changes the wallet_name was no longer supplied and so i believe we stopped the UI from displaying it. There is a ticket to get it re-instated at which point we will unskip these tests

Reply:
For now it only shows the key

Reply:
Think this will get it back https://github.com/vegaprotocol/vega/issues/6139